localnode/docker-compose.yml

```diff
@@ -235,9 +235,9 @@ services:
   # Hemi op-node
   op-node:
     # yamllint disable-line rule:line-length
-    image: "ghcr.io/hemilabs/op-node:ee580dd@sha256:c3c2bcabb0da02e6aca462a2756ef78d8a40e569281de1f4dac9a9eb8fa80359"
+    image: "ghcr.io/hemilabs/op-node:e79d444@sha256:c2757007112d81c2db1f0ad52cf12e1a3f2bf8313df434d9533abbf29b8b9d04"
```

Comment:
let's use the latest op-node, ghcr.io/hemilabs/op-node:9dc3004@sha256:fbbed98c8d9f8b1de1eb426b84b38c5df4fe2e04e9c757f4734dc1009d253b3e